exercise-1/README.md

```diff
@@ -71,27 +68,20 @@ Dump of assembler code for function vuln:
    0x080484fb <+25>:    ret
 End of assembler dump.
 ```
-So you might remember from [Intro 2](../intro-2) that you can overwrite values
-on the stack with a `strcpy()` vulnerability. In the lines of `main()` ,
-control is passed to the function`vuln()`. However, `vuln()` needs to know where to
-come back to in `main()` when it finishes. This is called a return address. In
-this case, `vuln()` should jump back to 0x0804851b, the instruction right after
-`main()` calls `vuln()`. When we get a SEGFAULT that we control, that means
-that we've overwritten the return address. What can we do with this? The
-possibilites are pretty much  endless. You have control over the code's flow,
-so maybe we can call some other function, namely `give_shell()`
+
+You might remember from [Intro 2](../intro-2) that you can overwrite values on the stack with a `strcpy()` vulnerability. In the lines of `main()`, control is passed to the function `vuln()`. However, `vuln()` needs to know where to return to in `main()` when it finishes. This is called a return address. In this case, `vuln()` should jump back to `0x0804851b`, the instruction right after `main()` calls `vuln()`. When we get a SEGFAULT that we control, that means that we've overwritten the return address. What can we do with this? The possibilities are pretty much endless. You have control over the code's flow, so maybe we can call some other function, namely `give_shell()`.
```

Comment:
added code formatting and collapsed into one paragraph as opposed to being split up arbitrarily